tests/test_environment_BondOrder.py

```diff
@@ -96,6 +96,35 @@ def test_repr(self):
         bo = freud.environment.BondOrder(1.5, 0, 12, 6, 6)
         self.assertEqual(str(bo), str(eval(repr(bo))))
 
+    def test_ref_points_ne_points(self):
+        lattice_size = 10
+        # big box to ignore periodicity
+        box = freud.box.Box.square(lattice_size*5)
+        angle = np.pi/30
+        ref_points, points = util.make_alternating_lattice(lattice_size, angle)
+
+        # actually not used
+        rmax = 1.6
+        k = 0
+
+        # to make sure what we skip ref_points ref_points neighbors
```

Comment:
I think this comment is outdated, you're not doing anything to avoid ref_points that are neighbors of other ref_points, are you? In case the comment is still relevant, there's a typo, what->that.